connector_importer/components/importer.py

```diff
@@ -63,9 +63,8 @@ class RecordImporter(Component):
     _record_handler_usage = "odoorecord.handler"
     _tracking_handler_usage = "tracking.handler"
     # a unique key (field name) to retrieve the odoo record
-    # if this key is an external/XML ID, set 'odoo_unique_key_is_xmlid' to True
+    # if this key is an external/XML ID, prefix the name with `xid::` (eg: xid::id)
     odoo_unique_key = ""
-    odoo_unique_key_is_xmlid = False
```

Comment:
Should we add a deprecated warning or an error if we set this class attribute? Not sure to remember to change this when migrating custom code.

Reply:
@sebalix fine for you with last commit? https://github.com/OCA/connector-interfaces/pull/98/commits/bed97993b3b82583b994c739f843b4d5959591b5

Reply:
@simahawk all good, thanks!